### PR TITLE
Initial upload of PCI for Windows JSON, and Microsoft app cleanup

### PR DESCRIPTION
**Pull request by danreichert**

### CODE REVIEW

File: Microsoft/PCI_Compliance_for_Windows_JSON/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# PCI Compliance for Windows JSON
+
+This is an updated version of the PCI Compliance for Windows app. It is updated to work with Windows Event Logs in the JSON formatted syntax. It is also utilizing the new Sumo Logic dashboard framework. 
+
+## Setup
+
+### Setup Collection
+
+This app only requires a standard Windows Event source(s) configured for the JSON-formatted event logs.
```
Comment (spchin) [inline Microsoft/PCI_Compliance_for_Windows_JSON/README.md:9]: Recommend adding a link to our help docs to set up the Windows Events source with JSON format.

File: Microsoft/PCI_Compliance_for_Windows_JSON/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+### Modify Dashboard JSON
+
+Prior to importing the app, find and replace all instances of _\_sourceCategory=windows/events_ with your appropriate metadata for your Windows Event logs.
+
+
+### Import App
+
+Import your modified version of the JSON file into Sumo Logic.
```
Comment (spchin) [inline Microsoft/PCI_Compliance_for_Windows_JSON/README.md:18]: Add a link to help docs for how to import the JSON.

Comment (spchin) [reply Microsoft/PCI_Compliance_for_Windows_JSON/README.md:18]: Might also be good to link to the existing help doc about these dashboards to describe what they are telling you.

**Review by spchin — CHANGES_REQUESTED** [review]
Minor changes suggested to the readme but other than that looks good.

**Comment by danreichert** [comment]
Suggested changes made to the README

**Pull request merged**